nextflow_schema.json

```diff
@@ -707,5 +707,12 @@
         {
             "$ref": "#/definitions/reference_genome_options"
         }
-    ]
+    ],
+    "properties": {
+        "preprocessing_qc_tool": {
+            "type": "string",
+            "default": "fastqc",
```

Comment:
missing description and help test?

I guess could be something long the lines of -:

Description: Specify which FASTQ quality control tool to use

Help: `falco` is aimed at being a drop-in replacement for FastQC but written in C++ for faster computation. We particularly recommend using falco when using long reads (due to reduced memory constraints), however is also applicable for short reads.